include/pf-applications/base/debug.h

```diff
@@ -49,28 +49,28 @@ namespace debug
 
   template <typename Vector>
   std::string
-  to_string(const Vector &vec)
+  to_string(const Vector &vec, std::string sep = ",")
```

Comment:
Is there a reason, we don't use `boost::algorithm::join`?

Reply:
Absolutely not. I will remove later these functions to `boost::algorithm::join`, I think it would be better to do that in a separate PR.